include/flatbuffers/flatbuffers.h

```diff
@@ -853,6 +882,16 @@ FLATBUFFERS_FINAL_CLASS
     if (string_pool) delete string_pool;
   }
 
+  /// @brief Get the underlying `vector_downward`
+  /// @return Returns the underlying `vector_downward` buffer
+  vector_downward &GetVectorDownward() { return buf_; }
```

Comment:
If we're going to be using inheritance, neither of these methods are necessary for access, and add unnecessary public functionality